ISLE/isleapp.cpp

```diff
@@ -540,6 +545,45 @@ void IsleApp::LoadConfig()
 
 	dictionary* dict = iniparser_load(iniConfig);
 
+	// [library:config]
+	// Load sane defaults if dictionary failed to load
+	if (!dict) {
+		SDL_LogInfo(SDL_LOG_CATEGORY_APPLICATION, "Loading sane defaults");
+		FILE* iniFP = fopen(iniConfig, "w");
+
+		if (!iniFP) {
+			SDL_LogError(SDL_LOG_CATEGORY_APPLICATION, "Failed to write config at '%s': %s", iniConfig, strerror(errno));
+			fclose(iniFP);
```

Comment:
No need for `fclose` if `iniFP` is `NULL`.

Also, attempting to write to the file shouldn't happen past this point.